lib/elixir/lib/kernel.ex

```diff
@@ -1384,6 +1384,11 @@ defmodule Kernel do
       iex> [1, 2, 3, 2, 1] -- [1, 2, 2]
       [3, 1]
 
+  It is important to note that `--/2` is right associative meaning:
```

Comment:
I think to clarify the example it should also show how it is interpreted with parentheses.